src/instructlab/train/linux_train.py

```diff
@@ -393,6 +394,7 @@ def model_generate(user, **kwargs):
 
     print("LINUX_TRAIN.PY: MERGING ADAPTERS")
     model = trainer.model.merge_and_unload()
-    model.save_pretrained("./training_results/merged_model")
+    model.save_pretrained(output_dir + "/merged_model")
```

Comment:
I don't think it matters in this case, but this is slightly preferred:

```suggestion
    model.save_pretrained(os.path.join(output_dir, "/merged_model"))
```

Reply:
I would have applied this, but looks like maintainer access to the PR branch was turned off